content/en/blog/introduction_to_wamr_running_modes/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Introduction to WAMR running modes"
+description: ""
+excerpt: ""
+date: 2023-01-19T17:20:07+08:00
+lastmod: 2023-01-28T17:20:07+08:00
+draft: false
+weight: 50
+images: []
+categories: ['introduction']
+tags: []
+contributors: ['Tianlong Liang']
+pinned: false
+homepage: false
+---
+
+When you first meet WAMR, want to give it a taste, either using `iwasm` to run a hello world wasm application, or embed a WAMR into your hello world program, I bet you won't even sweat a bit. But when you're up to do something serious, maybe use it for real world applications and start to think about performance and response time, things changes. At this time, the different running modes (AOT, interpreter, fast interpreter, fast JIT, LLVM JIT, multi-tier JIT) we proudly made, aiming to provide user more adaptability and flexibility, could backfire and encumber user.
+
+Although we try to make CMake cache variables' name(CMake cache variables serve as the method to choose running mode at compile time) as explanatory as possible, because our background varies and the same term may mean differently to us, eventually, it can still cause confusion. And when it did, you may have to go back and forth from the documentation page to your develop environment, trying to understand each running mode are for, compile and test WAMR for your wasm app, see how does each running mode perform. We understand it's a chore (even worse, it maybe a dealbreaker for some people) to go through this process.
+
+To address this potential problem, and to build common understanding of definition toward each running modes, I wrote this blog, hopefully, it could fasten your learning process to pick out the running modes of WAMR serve you best, and make your day just easier.
+
+## Why? Why so many running modes?
+
+Our goal is to provide user the adaptability and flexibility. User could choose one to achieve the best performance and resource usage according to deploy environment and nature of application. But as more running mode supported over the years, the learning curve for new user now is a little unfriendly. But let's go through this blog together, I believe when you finish reading it, you would understand our intention to develop each running mode, and feel confident to choose the one (or combination) you need.
```

Comment:
`User could` -> `User can`?
`But as more running mode supported over the years` -> `But as more running mode were supported over the years`